dime-coding-standards/checklists/tex/peer_code_review_guidance_note.tex

```diff
@@ -0,0 +1,131 @@
+
+	\section*{Overview}
+ Every quarter, DIME Analytics organizes the \textbf{Peer Code Review}, which is a real-time code and data-quality assurance process. This is a structured opportunity for participants to exchange, run, and provide feedback on each other’s code. Apart from increasing reproducibility, transparency and adherence to best practices, it is a great learning opportunity - code writers get to look at their work from a different perspective, and reviewers are exposed to different styles and practices. 
+\hfill \break
+\begin{flushleft} The peer code review is designed for scripts that are modular enough to be understood on their own (a reviewer can understand the script without having to refer to other project code files) and are short enough for a reviewer to read through in approximately half a day (depending on complexity). We recommend submitting recently-completed tasks, to allow corrections to be made in real time.
+
+\section*{How it works}
+
+	\begin{enumerate}
+	    
+	  
+		\setlength\itemsep{-0.1em}
+		\item Teams \href{https://survey.wb.surveycto.com/collect/code_review_sign_up?caseid=}{sign-up} to participate in the code review each quarter. All DIME Research Assistants working on code are expected to participate, and any Bank staff or consultant who would like to get feedback on their code is welcome to participate.
+    
+
+		\item All participants prepare a code package to submit to review, following \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Peer%20Code%20Review%20Submission%20Checklist.pdf}{these guidelines}. Submitted code should be \textbf{no longer than 1000 lines}. 
+        
+        \begin{itemize}
+            \item As part of this step, teams will identify what tasks they want review (data cleaning, construction, or analysis) and whether they want the reviewer to assess  \textbf{computational reproducibility}. 
+            \item \textbf{Note} In order for the reviewer to assess reproducibility, the peer review submission package must include a \textbf{de-identified} version of the dataset.
+        \end{itemize}
+
+        \item Participants are paired for review based on preferred statistical software and length of code.
+
+        \item The actual peer code review activity takes place over the course of one week, which includes:
+         \begin{itemize}
+            \item A \textbf{kickoff session} with an overview of the process, and pair assignments. 
+            \item A \textbf{group work session} with technical support from DIME Analytics. Pairs work together, and answer questions to allow each other to review their submitted code.
+        \end{itemize}
+
+       \item Reviewers review the code using structured checklists:
+        \begin{itemize}
+            \item \textbf{Mandatory}: \href{https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/Reviewer%20Feedback%20Checklist.pdf}{Reviewer Feedback Checklist}
```

Comment:
Given that reproducibility check isn't mandatory, i think we want to remove that from the reviewer feedback sheet and create a separate checklist (the same way we have cleaning, construction, etc).

Reply:
That sounds good, I'll make those changes to the checklists. And I also agree on the point about re-organizing the folder structure. I'll create a new one for this.